src/datomic_rest_api/rest/gene.clj

```diff
@@ -2191,34 +2226,56 @@
                :in $ ?gene
                :where [?vh :variation.gene/gene ?gene]
                       [?var :variation/gene ?vh]
-                      (not [?var :variation/allele _])]
+                      (not [?var :variation/allele _])
+                      (not [?var :variation/phenotype _])]
              db (:db/id gene))
           (map #(process-variation (entity db %))))
-     :description "polymorphisms and natural variations contained in the strain"}))
+     :description "polymorphisms and natural variations currently with no associated phenotype"}))
+
+(defn- alleles-count [gene]
```

Comment:
I think the duplication here could be factored out. not critical for merge, perhaps we can pair on this later.